### ISSUE TITLE
Create Home/Overview screen for Membership account

### ISSUE THREAD

**Issue by Lenorette** [Joystream/atlas#1839]
Create additional Tab in Membership account designs with Home/Overview screen.
In which user (owner) will be able to manage NFTs that are currently on sale and there is some action to be done:

1. Auctions I'm participating in right now (eg. to see if I've been outbid),
2. Auctions that I lost in which I can withdraw my tokens,
3. Auctions I won that is pending settlement.

**Comment by dmtrjsg** [comment]
@toiletgranny are there any use cases that are not very well covered with what we have right now?

**Comment by toiletgranny** [comment]
@dmtrjsg Yes, if I remember correctly, we're missing a view in which, as a member, I can have a quick overview of:
1. Auctions I'm participating in right now (eg. to see if I've been outbid),
2. Auctions that I lost in which I can withdraw my tokens,
3. Auctions I won that is pending settlement.

Currently, the only way to keep track of this is using notifications, which is not ideal since they can be mixed with other kinds of notifications, and might not reflect the current state (if I settle an auction, I would still see a past notification prompting me to do so).
